engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -127,11 +127,15 @@ public void swapReadWriteBuffers() {
         buffersSwapped = !buffersSwapped;
     }
 
-    public FBO getPrimaryBufferUrn() {
+    public FBO getPrimaryBuffer() {
```

Comment:
Had a look and I now understand better your line of thinking. Let's see where that leads us.